### PR TITLE
add code pen link to atlas-css readme

### PR DESCRIPTION
**Pull request by wibjorn**

### CODE REVIEW

File: css/tokens/README.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 A design tokens is a definition of a design-related variable, such as one that affects color, typography, or spacing. Design tokens must be shareable and fairly platform agnostic. Atlas's design tokens are written in Scss (buzzword: source of truth!), and though a [build process](../tokens/index.js), translated and published into JSON so they can be more easily shared by solutions that do not use Scss.
 
+## Use atlas-css on Code Pen
+
+[Using CodePen as a sandbox](https://codepen.io/wibjorn/pen/JjEzrQN), you can try out the framework.
+
```
Comment (ollips) [inline css/tokens/README.md:8]: Move to the `css/README.md` :)

**Review by ollips — CHANGES_REQUESTED** [review]

File: css/tokens/README.md (hunk 1 above)
Comment (wibjorn) [reply css/tokens/README.md]: 👍 updated

File: css/README.md [hunk 2]
```diff
@@ -2,6 +2,13 @@
 
 Welcome to the Atlas CSS framework, the code backing the Atlas design system. This framework is built and maintained by the Developer Relations web team for use across Microsoft.
 
+## Use atlas-css on Code Pen
+
+[Using CodePen as a sandbox](https://codepen.io/wibjorn/pen/JjEzrQN), you can try out the framework.
+
+## Installation
+
+
```
Comment (ollips) [inline css/README.md:11]: nit: remove one empty line :)

**Review by ollips — CHANGES_REQUESTED** [review]

File: css/README.md (hunk 2 above)
Comment (wibjorn) [reply css/README.md:11]: arg! indeed

Comment (wibjorn) [reply css/README.md:11]: done

**Review by ollips — APPROVED** [review]

**Pull request merged**